ironmon_tracker/Input.lua

```diff
@@ -180,6 +231,42 @@ function Input.checkJoypadInput()
 	end
 end
 
+function Input.infoShortcutPressed()
+	-- At current, don't activate if inside a battle or in menus (Select in-game reorders stuff) or using MGBA
+	if Battle.inActiveBattle() or Program.isInStartMenu() or not Main.IsOnBizhawk() then
+		return
+	end
```

Comment:
is the in-battle check needed now that it's R and not SELECT? (L & R iirc do swap tabs while in the bag so may still want the in-start-menu check)

Reply:
Good call, I think i can remove the in-battle check and the in-start menu check. To my knowledge, the L and R buttons have no impact there within the game itself.

I will add another condition to display a more contextual info screen in that case.

Reply:
Just double-checked and L and R function like left/right directional buttons while in menus like the bag or on the pokemon summary screen so probably want to leave the in-start-menu check in there

in-battle they don't directly do something unless you go into the same menus (bag, pokemon, summary screen) it seems, unless there's a doubles functionality or something